app/modules/intelligence/agents/chat_agents/code_gen_chat_agent.py

```
@@ -70,6 +77,7 @@ async def run(
             "node_ids": node_ids,
         }
         graph = self._create_graph()
+        llm = await self._get_llm()
```

Ruff (0.8.2)

Local variable `llm` is assigned to but never used

Remove assignment to unused variable `llm`

(F841)